elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AsyncProperties.java

```diff
@@ -11,7 +11,7 @@
  * Extra properties for setting up async query support.
  */
 @Data
-public class AsyncProperties extends ControllerProperties {
+public class AsyncProperties {
```

Comment:
Since AsyncProperties does not have use for `path` property, removed extends ControllerProperties.